### ISSUE TITLE
Add usage example to the readme

### ISSUE THREAD

**Issue by tomsoderlund** [vikiival/vercelgl#8]
Hi @vikiival, do you have a usage example of this project?

I'm trying

    curl -X POST -H 'Content-type: application/json' --data '{"url":"https://www.tomorroworld.com/"}' https://vikiival-vercelgl.vercel.app/

and get a `405` (Method Not Allowed) back, no JSON response.

**Comment by vikiival** [comment]
```bash
curl --request POST \
  --url http://<base_url>.vercel.app/api/screenshot \
  --header 'Content-Type: application/json' \
  --data '{
  "url": "https://www.tomorroworld.com/"
}'
```

I got timeout, your page takes too long to respond (more than 10s)

<img width="664" alt="Screenshot 2024-01-18 at 13 12 03" src="https://github.com/vikiival/vercelgl/assets/22471030/ab59c9b2-9e3e-446d-b443-8a982a3457f8">

**Comment by tomsoderlund** [comment]
Ah, I was missing the `/api/screenshot` route. Here I try with another `url` parameter:

    curl --request POST \
      --url https://vikiival-vercelgl.vercel.app/api/screenshot \
      --header 'Content-type: application/json' \
      --data '{"url":"https://www.google.com/"}'

But I get function timeout after 15 seconds – do you have to configure `maxDuration` (https://vercel.com/docs/functions/configuring-functions/duration) on Vercel?

**Comment by tomsoderlund** [comment]
I bumped up `maxDuration` to 60 seconds, but got this after 41 seconds:

    TimeoutError: Waiting for selector `canvas` failed: Waiting failed: 30000ms exceeded

**Comment by vikiival** [comment]
> I bumped up maxDuration to 60 seconds, but got this after 41 seconds:

I have very specific case :) that is just doing sceenshot of canvas on the page

**Comment by vikiival** [comment]
Closed by [b0fc7e7](https://github.com/vikiival/vercelgl/commit/b0fc7e7e2dc0b9fb407de1a853ba063871640af4)

**Comment by vikiival** [comment]
Thank you for opening the issue!
